test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -19,42 +19,44 @@ public class HelloWorldTests
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
-        //[InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:5061/", ServerType.Kestrel)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:5062/", ServerType.Kestrel)]
-        //[InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, "http://localhost:5063/", ServerType.Kestrel)]
-        [InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:5064/", ServerType.Kestrel)]
-        public Task HelloWorld_IISExpress(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, string applicationBaseUrl, ServerType delegateServer)
+        //[InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:5061/", ServerType.Kestrel, ApplicationType.Standalone)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:5062/", ServerType.Kestrel, ApplicationType.Standalone)]
```

Comment:
This is a little misleading since Standalone CLR app isn't quite right. The reason for this is that for this particular repo, I'm using `AppliationType` to determine whether to use the `TestApp` or `TestApp.Portable`. The published CLR app uses the same `web.config` as the standalone CoreCLR app, launching via `.\TestApp.exe` whereas the porable CoreCLR app launches via `dotnet .\TestApp.Portable.dll`. For other repos, we can rely on the publish-iis tool to do the correct thing and won't have this confusion.